.github/workflows/publish.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 8.x, 10.x, 11.x ]
+        node-version: [ 12.x, 13.x, 14.x, 15.x ]
```

Comment:
I wouldn't change this just yet as this might break the release flow. Currently this is even with what we had with travis.

Changing this and `npm install` in the following comment might need to be tested for proper publishing.

It would be better to have a ticket for upgrading the node-versions and have this in a separate PR.

Reply:
The thing is:
- Those old Node version are no longer under maintenance. 
- Replacing the `npm install` with `yarn install` can have multiple dependency consequences, since there's no `yarn.lock` file.

Reply:
I see that which is why I suggest we have a ticket for this and properly test any dependency issues that might arise by changing this. I remember there were dependency  issues when I changed node versions and the `yarn install `to `npm install`.

The current yml works without issues so to change it I would say we do it as part of a separate ticket an not this PR.

Reply:
Resolved by testing this PR on the test npm account 👍